products/internetexplorer.md

```diff
@@ -0,0 +1,50 @@
+---
+permalink: /internetexplorer
+title: Internet Explorer
+releasePolicyLink: https://docs.microsoft.com/en-us/lifecycle/faq/internet-explorer-microsoft-edge#what-is-the-lifecycle-policy-for-internet-explorer-
+releaseDateColumn: true
+releaseColumn: false
+iconSlug: internetexplorer
+activeSupportColumn: false
+eolColumn: Security and technical support
+sortReleasesBy: releaseDate
+releases:
+
+-   releaseCycle: "7" 
+    eol: 2023-10-10
+    releaseDate: 2006-10-18
+
+-   releaseCycle: "8"
+    eol: 2016-01-12
+    releaseDate: 2009-06-17
+
+    
+-   releaseCycle: "9"
+    eol: 2016-01-12
+    releaseDate: 2011-03-15
+
+-   releaseCycle: "10"
+    eol: 2020-01-31
+    releaseDate: 2012-10-30
+
+-   releaseCycle: "11 (Semi-Annual Channel)" # This applies to 99% of users
+    eol: 2022-06-14
```

Comment:
```suggestion
    eol: 2022-06-15
```

Reply:
Date on MS documentation is the 14th, I believe the wording is that it is end of life on the 14th and that the 15th is the first day it is end of life. 

https://docs.microsoft.com/en-us/lifecycle/products/internet-explorer-11-on-windows-10

Reply:
Matching upstream seems more accurate.

Reply:
> Matching upstream seems more accurate.

You mean having the 14th or 15th here? Because announcements from Microsoft usually say the 15th, while the documentation says the 14th.

Reply:
14th, as on the table.